src/Test/Perf/util/RelativeDirectory.cs

```diff
@@ -23,27 +23,13 @@ public RelativeDirectory([CallerFilePath] string workingFile = "")
 
         public string MyWorkingDirectory => _workingDir;
 
-
-        /// Returns the directory that you can put artifacts like
-        /// etl traces or compiled binaries
-        public string MyArtifactsDirectory
-        {
-            get
-            {
-                var path = Path.Combine(MyWorkingDirectory, "artifacts");
-                Directory.CreateDirectory(path);
-                return path;
-            }
-        }
-
-        public string MyTempDirectory
+        public string TempDirectory
         {
             get
             {
-                var workingDir = MyWorkingDirectory;
-                var path = Path.Combine(workingDir, "temp");
-                Directory.CreateDirectory(path);
-                return path;
+                var tempDirectory = Path.Combine(Environment.ExpandEnvironmentVariables("%SYSTEMDRIVE%")+@"\", "PerfTemp");
+                Directory.CreateDirectory(tempDirectory);
```

Comment:
Currently I am not cleaning it up but I am thinking if we should. Would we run into a situation where we need to take a look into these(like weird behavior in test runs, broken test solution, or something I cannot think of)

What do you think?